source/includes/authorization_configuration/_10-index.md.erb

```diff
@@ -0,0 +1,57 @@
+## Get all authorization configurations
+
+```shell
+$ curl 'https://ci.example.com/go/api/admin/security/auth_configs' \
+      -u 'username:password' \
+      -H 'Accept: <%= data.apis.versions.authorization_configurations %>'
+```
+
+> The above command returns JSON structured like this:
+
+```http
+HTTP/1.1 200 OK
+Content-Type: <%= data.apis.versions.authorization_configurations %>; charset=utf-8
+ETag: "32628084ef10b0c82a55f8f2f867d8d0"
+```
+
+```json
+{
+  "_links": {
+    "self": {
+      "href": "https://ci.example.com/go/api/admin/security/auth_configs"
+    },
+    "doc": {
+      "href": "https://api.gocd.io/#authorization-configuration"
+    },
+    "find": {
+      "href": "https://ci.example.com/go/api/admin/security/auth_configs/:auth_config_id"
+    }
+  },
+  "_embedded": {
+    "auth_configs": [
+      {
```

Comment:
Shouldn't each auth config give the self and find links? At least that's what the other index calls' responses have.